src/plugins/recordTypes/organization/forms/mini.jsx

```diff
@@ -0,0 +1,29 @@
+const template = (pluginContext) => {
+  const {
+    React,
+  } = pluginContext.lib;
+
+  const {
+    Field,
+  } = pluginContext.recordComponents;
+
+  return (
+    <Field name="document">
+      <Field
+        name="relation-list-item"
+        subpath="rel:relations-common-list"
+        showChildren={false}
+        showSiblings={false}
+      />
+
+      <Field name="foundingPlace" />
+      <Field name="foundingDateGroup" />
```

Comment:
Try putting foundingPlace and foundingDateGroup in a row. I think there's enough room.